src/content/ja/updates/2019/02/rendering-on-the-web.md

```diff
@@ -1,12 +1,7 @@
-project_path: /web/_project.yaml
-book_path: /web/updates/_book.yaml
+project_path: "/web/_project.yaml"
+book_path: "/web/updates/_book.yaml"
 
-{# wf_updated_on: 2019-02-06 #}
-{# wf_published_on: 2019-02-06 #}
-{# wf_tags: fundamentals, performance, app-shell #}
-{# wf_featured_image: /web/updates/images/2019/02/rendering-on-the-web/icon.png #}
-{# wf_featured_snippet: Where should we implement logic and rendering in our applications? Should we use Server Side Rendering? What about Rehydration? Let's find some answers! #}
-{# wf_blink_components: N/A #}
+{# wf_updated_on: 2019-08-27 #} {# wf_published_on: 2019-02-06 #} {# wf_tags: fundamentals, performance, app-shell #} {# wf_featured_image: /web/updates/images/2019/02/rendering-on-the-web/icon.png #} {# wf_featured_snippet: Where should we implement logic and rendering in our applications? Should we use Server Side Rendering? What about Rehydration? Let's find some answers! #} {# wf_blink_components: N/A #}
```

Comment:
@ilyaspiridonov It's still removing line-breaks
https://github.com/alconost/gitlocalize-google/issues/4

Reply:
@agektmr Trying to find a solution at the moment.